# LuxOS: bez podpisu .asc (kontrolni soucet SKIP; stavi se bez site a bez klicu) - archiv je zapichnuty sha256
--- a/PKGBUILD
+++ b/PKGBUILD
@@ -14,11 +14,9 @@
 url='http://www.libpng.org/pub/png/libpng.html'
 license=('custom')
 depends=('glibc' 'zlib' 'gcc-libs')
-source=("https://sourceforge.net/projects/libpng/files/libpng-${pkgver}.tar.xz"{,.asc}
+source=("https://sourceforge.net/projects/libpng/files/libpng-${pkgver}.tar.xz"
         "https://sourceforge.net/projects/libpng-apng/files/libpng12/${pkgver}/libpng-${pkgver}-apng.patch.gz")
-validpgpkeys=('8048643BA2C840F4F92A195FF54984BFA16C640F') # Glenn Randers-Pehrson
 sha256sums=('b4635f15b8adccc8ad0934eea485ef59cc4cae24d0f0300a9a941e51974ffcc7'
-            'SKIP'
             '281fd5f0165762967a18302dca217de3212be4a3437f95805be44f1ac9db1a5d')
 
 prepare() {
